CONTRIBUTING.md

```diff
@@ -1,14 +1,14 @@
 # Contributing
 
-We'd love you to contribute to [dyrector.io](https://dyrector.io/) platform. We want contribution to dyrector.io to be fun, enjoyable, and educational for anyone and everyone. All contributions are welcome, including issues, new docs as well as updates and tweaks, blog posts, workshops, and more.
+We would love you to contribute to [dyrector.io](https://dyrector.io/) and help make it better! We want contributing to Dyrector to be fun, enjoyable, and educational for anyone and everyone. All contributions are welcome, including issues, and new docs, as well as updates and tweaks, blog posts, workshops, and more.
```

Comment:
Dyrector is not the valid name, dyrector.io is the valid one, please do not change this.